pkg/preparer/setup.go

```diff
@@ -404,6 +404,14 @@ func New(preparerConfig *PreparerConfig, logger logging.Logger) (*Preparer, erro
 		return nil, util.Errorf("Could not create preparer pod directory: %s", err)
 	}
 
+	// Artifact files are downloaded to os.TempDir().
+	// Since we extract artifact files as target user, we must allow them to access the tmpdir.
+	// We expect that there is no sensitive information in TempDir, so 0755 is safe, though 0711 could be considered.
+	err = os.Chmod(os.TempDir(), 0755)
```

Comment:
Eh. I'm going to do it in the downloader. Interesting test interactions ("operation can't be nil"), if I have it here, I wonder what that means